packages/terra-framework-docs/src/terra-dev-site/doc/embedded-content-consumer/example/BasicConsumer.jsx

```diff
@@ -4,14 +4,10 @@ import { Consumer } from 'xfc';
 import '../providers/EmbeddedContentConsumerCommon.module.scss';
 
 Consumer.init();
-
```

Comment:
nit: can we keep the newline here or remove it from the other doc files as well?